module/VuFindSearch/src/VuFindSearch/Backend/Summon/PsrConnector.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace VuFindSearch\Backend\Summon;
+
+use GuzzleHttp\Client as HttpClient;
+use Psr\Log\LoggerAwareInterface;
+use Psr\Log\LoggerInterface;
+use SerialsSolutions_Summon_Exception;
+
+/**
+ * PSR-compliant port of VuFindSearch\Backend\Summon\PsrConnector
```

Comment:
I think this comment should reference the original SerialsSolutions\Summon\Laminas connector -- I bet it got changed in a global search-and-replace.